scripts/pythran

```diff
@@ -132,7 +126,7 @@ parser.add_argument('-O', dest='extra_Oflags', metavar='level',
                     action='append',
                     help='any optimization level relevant to the underlying '
                     'C++ compiler',
-                    default=['2'])
+                    default=list())
```

Comment:
On Fri, May 22, 2015 at 07:14:34AM -0700, pbrunet wrote:
> > @@ -132,7 +126,7 @@ parser.add_argument('-O', dest='extra_Oflags', metavar='level',
> >                      action='append',
> >                      help='any optimization level relevant to the underlying '
> >                      'C++ compiler',
> > -                    default=['2'])
> > +                    default=list())
> 
> Default is O0?

default is set by distutils to O2